src/include/log.h

```diff
@@ -351,8 +351,9 @@ struct __wt_log_desc {
                          * MongoDB to detect users accidentally running old binaries on a newer
                          * release. There are no actual log file format changes with version 2 and
                          * 3.
+                         * NOTE: We bumped the log version from 3 to 4 for the same reason.
```

Comment:
I would suggest adding this in some form to the previous sentence. Something like:
"There are no actual log file format changes with version 2, version 3 and version 4."

Reply:
Done.